scripts-components/in-view/README.md

```diff
@@ -0,0 +1,30 @@
+# InView
+
+The manGoweb InView component. Watches elements entering and leaving viewport.
+
+## Installation
+
+`$ npm install @mangoweb/in-view`
+
+Works best with [`@mangoweb/scripts-base`](https://www.npmjs.com/package/@mangoweb/scripts-base).
```

Comment:
Jak jako „works best“? Vždyť je to hard dependency, ne?

Reply:
Závislost je na `class Component` a `ComponentInitializationError`.

To  „works best“ je myšleno jako že nejlépe funguje `initializeComponents`, na které přímou závislost nemám. `initializeComponents` by mělo být asi jako peer dependency, kdy to nebylo ve stejném balíčku jako `Component` a `...Error`.

Reply:
Podle mě neexistuje žádný plán vyčlenit `initializeComponents` někam do separátního balíčku. Vidíš pro to jakýkoliv důvod?

Reply:
Nevidím. Ale tady je spíš otázka, jestli nemůže vzniknout alternativa k `initializeComponents`. Teoreticky ti nic nebrání `initializeComponents` vůbec nepoužít.

Hele, klidně tu větu můžu smazat. Šlo mi jen o to někam dostat odkaz na `scripts-base`. :D

Reply:
Upraveno.